integrationTests/state/stateTrieSync/stateTrieSync_test.go

```diff
@@ -53,7 +53,8 @@ func createTestProcessorNodeAndTrieStorage(
 		TrieStore:            mainStorer,
 		GasScheduleMap:       createTestGasMap(),
 	})
-	_ = node.Messenger.CreateTopic(common.ConsensusTopic+node.ShardCoordinator.CommunicationIdentifier(node.ShardCoordinator.SelfId()), true)
+	_ = node.MainMessenger.CreateTopic(common.ConsensusTopic+node.ShardCoordinator.CommunicationIdentifier(node.ShardCoordinator.SelfId()), true)
+	_ = node.FullArchiveMessenger.CreateTopic(common.ConsensusTopic+node.ShardCoordinator.CommunicationIdentifier(node.ShardCoordinator.SelfId()), true)
```

Comment:
not required

If this doesn't panic, do we need to close all messengers on the defer functions? (valid on all occurrences) 
Maybe create a function on the test processor node that will close all messengers?

Reply:
the test processor node already has a Close() method that closes all messengers.. updated tests to call it